docs/content/usage/using-with-kubernetes.md

```diff
@@ -0,0 +1,9 @@
+---
+weight: 3
+---
+
+The default network plugin for `cri-dockerd` is set to `cni` on Linux. To change this, `--network-plugin=${plugin}`
```

Comment:
How about putting the steps point by point?

Reply:
The tricky part here is that it's actually only a couple steps but worded badly. This is something from the original docs. I'll see if I can make it easier to understand.